get_logs.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/bash
```

Comment:
I think we should move utility scripts to different folder. Maybe named as utils.